packages/aws-cdk-lib/aws-ecs/test/fargate/fargate-task-definition.test.ts

```diff
@@ -21,28 +21,24 @@ describe('fargate task definition', () => {
 
     });
 
-    test('support lazy cpu and memory values', () => {
+    test('does not support lazy cpu and memory values', () => {
```

Comment:
We should be able to support this by adding validation on the `node` (like [this](https://github.com/aws/aws-cdk/blob/4e9ef15a0cf8b1bc664b5d22b34af9eaaa17bec8/packages/aws-cdk-lib/aws-ecs/lib/base/task-definition.ts#L524)):
```ts
this.node.addValidation({ validate: () => this. validateFargateTaskDefinitionMemoryCpu(...);
```